src/styles/about.scss

```diff
@@ -0,0 +1,98 @@
+.about {
+  font-size: 14px;
+  margin-bottom: 110px;
+
+  @include tablet {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-gap: 30px;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    grid-template-columns: repeat(12, 1fr);
+    position: relative;
+    margin-bottom: 200px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__description {
+    @include tablet {
+      grid-column-start: 1;
+      grid-column-end: 4;
+    }
+
+    @include desktop {
+      padding-left: 100px;
+      grid-column-start: 1;
+      grid-column-end: 6;
+    }
+  }
+
+  &__paragraph {
+    font-size: 14px;
+    max-width: 500px;
+    margin-bottom: 30px;
+
+    @include desktop {
+      max-width: 380px;
+      // text-align: justify;
+    }
```

Comment:
There seems to be commented code that isn't being used. It's generally a good practice to remove such code to keep the codebase clean.